app/src/main/res/values/strings.xml

```diff
@@ -238,7 +238,7 @@
     <!-- This is the title of the device security dialog box. -->
     <string name="secure_your_device">Secure your device.</string>
```

Comment:
Should this be title-case? Honest question.

Reply:
The design is intentionally a sentence and sentence case

<img width="224" alt="Screen Shot 2019-05-08 at 12 51 54 PM" src="https://user-images.githubusercontent.com/49511/57400185-2e729000-7190-11e9-93f3-6fe8ae0844d2.png">

I'll let @changecourse chime in and _course_ correct us if we should instead do a heading/title case instead... but will assume not for now and mark "resolved"